src/libcore/num/mod.rs

```diff
@@ -245,42 +245,54 @@ macro_rules! int_impl {
      $reversed:expr, $le_bytes:expr, $be_bytes:expr,
      $to_xe_bytes_doc:expr, $from_xe_bytes_doc:expr) => {
         doc_comment! {
-            concat!("Returns the smallest value that can be represented by this integer type.
+            concat!("The smallest value that can be represented by this integer type.
 
 # Examples
 
 Basic usage:
 
 ```
-", $Feature, "assert_eq!(", stringify!($SelfT), "::min_value(), ", stringify!($Min), ");",
+", $Feature, "assert_eq!(", stringify!($SelfT), "::MIN, ", stringify!($Min), ");",
```

Comment:
To fix the latest CI failure, you have to add `#![feature(assoc_int_consts)]` somewhere here. The doc comments also need to specify which unstable features they use.

Reply:
Oh. Is that where those came from. I failed to realize the documentation examples does not compile as the libcore/libstd crates and thus need their own. Thanks!